js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/EvaluationConditionalLabel.tsx

```diff
@@ -4,15 +4,139 @@ import {
   Code,
   Colors,
   FontFamily,
+  Icon,
   MiddleTruncate,
   Tooltip,
 } from '@dagster-io/ui-components';
+import {useContext} from 'react';
 import styled from 'styled-components';
 
+import {EvaluationHistoryStackItem} from './types';
+import {NewEvaluationNodeFragment} from './types/GetEvaluationsQuery.types';
+import {TimeContext} from '../../app/time/TimeContext';
+import {DEFAULT_TIME_FORMAT} from '../../app/time/TimestampFormat';
+import {timestampToString} from '../../app/time/timestampToString';
+
 interface Props {
   segments: string[];
 }
 
+export const EvaluationSinceLabel = ({
+  evaluation,
+  pushHistory,
+}: {
+  evaluation: NewEvaluationNodeFragment;
+  pushHistory?: (item: EvaluationHistoryStackItem) => void;
+}) => {
+  const {
+    timezone: [userTimezone],
+    hourCycle: [userHourCycle],
+  } = useContext(TimeContext);
+
+  const locale = navigator.language;
+  const {expandedLabel, sinceMetadata, entityKey} = evaluation;
+
+  if (!sinceMetadata || expandedLabel.length !== 3 || expandedLabel[1] !== 'SINCE') {
+    throw new Error(
+      'sinceMetadata must be set and expandedLabel must be of the form [condition, SINCE, condition]',
+    );
+  }
+
+  if (!sinceMetadata.triggerTimestamp || !sinceMetadata.resetTimestamp) {
+    throw new Error('sinceMetadata must have both triggerTimestamp and resetTimestamp set');
+  }
+
+  const [triggerCondition, _operator, resetCondition] = expandedLabel;
+  const triggerLabel = triggerCondition?.slice(1, -1) || ''; // remove parentheses
+  const resetLabel = resetCondition?.slice(1, -1) || ''; // remove parentheses
+  const triggerTime = timestampToString({
+    timestamp: {unix: sinceMetadata.triggerTimestamp},
+    locale,
+    timezone: userTimezone,
+    timeFormat: DEFAULT_TIME_FORMAT,
+    hourCycle: userHourCycle,
+  });
+  const resetTime = timestampToString({
+    timestamp: {unix: sinceMetadata.resetTimestamp},
+    locale,
+    timezone: userTimezone,
+    timeFormat: DEFAULT_TIME_FORMAT,
+    hourCycle: userHourCycle,
+  });
```

Comment:
Is it possible to use `useFormatDateTime`​? It encapsulates the timezone and hourCycle settings.